build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -255,10 +287,12 @@ gpgcheck=0
 EOF
 
 if [ $ostype == "CentOS" ]; then
```

Comment:
simp config does this for us, using grep "\\[*\\]" *CentOS*.repo | cut -d "[" -f2 | cut -d "]" -f1 | xargs yum-config-manager --disable
* Why are we doing this in two places
* Why are we doing this differently?